packages/cli-kit/assets/cli-ruby/lib/shopify_cli/constants.rb

```diff
@@ -67,7 +68,7 @@ module EnvironmentVariables
     module SupportedVersions
       module Ruby
         FROM = "2.7.5"
```

Comment:
I guess 2.7.0 should also work, no?

```suggestion
        FROM = "2.7.0"
```

Reply:
not sure why, but I noticed that it was 2.7.5 in our TS code as well, maybe for some security fixes?

Reply:
It was updated [here](https://github.com/Shopify/shopify-cli/pull/2572) when dropping support for 2.6 (approved by me 😅). But there's no explanation... @charlespwd do you remember if there's any good reason?